### ISSUE TITLE
trim interval core dumped error

### ISSUE THREAD

**Issue by avilella** [nanoporetech/dorado#539]
Running 0.5.0+0d932c0 on a dataset produced by a MinION flowcell running a version of MinION that's a few months old.

```
I transformed the fast5/ into pod5 with the command below:

$ pod5 convert fast5 ./fast5/*.fast5 --output output_pod5s/ --one-to-one ./fast5/

Then ran dorado like so:

$ dorado basecaller --device cuda:0 $HOME/dorado/dna_r10.4.1_e8.2_400bps_sup@v4.1.0 output_pod5s/ > scalls.bam
[2023-12-18 22:04:36.245] [info] > Creating basecall pipeline
[2023-12-18 22:07:35.872] [info]  - set batch size for cuda:0 to 640
terminate called after throwing an instance of 'std::invalid_argument'
  what():  Trim interval 133-127 is invalid for sequence ATCTATTCGTTCAGTTCATATGCTTCAGCTTTCTGTTGGTGCTGATATTGCAAGCAGTGGTATCAACGCAGAGTACTTTTTTTTTTTTTTTTTTTTTTTTTTTTTTCTGTACTTCGTTCAGTTACGTATTGCTCGCTCTATCTTCCTACACGACGCTCTTCCGATCTTACGGTAAGACCCACCCCCAAAAAAAAAAAAAAAAAAAAAAAAAGTACTCTGCGTTGATACCACTGCTTGCAATATCAGCACCGGGGT
Aborted (core dumped)
```

Any ideas?

**Comment by malton-ont** [comment]
Hi @avilella. This looks to be the same problem as in https://github.com/nanoporetech/dorado/issues/529. The current workaround is to basecall with the `--no-trim` option - see https://github.com/nanoporetech/dorado/issues/529#issuecomment-1858410826.

**Comment by tijyojwad** [comment]
Hi @avilella - this should be fixed in v0.5.1 - https://github.com/nanoporetech/dorado#installation
